### DISCUSSION TITLE
Windows scrobbler

### DISCUSSION THREAD

**Discussion by erikt311** [Taxel/PlexTraktSync#614] [Support] [answered]
Could somebody clarify some basic things to me? I’m fairly technical, but not much python for Windows experience.  

I installed (Win10, latest updates, latest python for Win) with the .bat file, understand the configuration, etc. First sync seems to be going fine. 

To get scrobbling to work on windows, can I just create a Powershell script or bat file or something to run the watch command indefinitely? Or is there something else I need to do?

I’m OK just sticking with a scheduled sync if need be, but getting it working real-time would be great!

**Comment by xiongster** [comment, marked as answer]
Can you be more specific? What do you mean by 'getting it working real-time'?
